SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -285,7 +287,7 @@ bool USpatialClassInfoManager::ShouldTrackHandoverProperties() const
 
 	const UAbstractLBStrategy* Strategy = NetDriver->LoadBalanceStrategy;
 	check(Strategy != nullptr);
-	return Strategy->RequiresHandoverData() || Settings->bEnableHandover;
+	return Strategy->RequiresHandoverData() || USpatialStatics::IsHandoverEnabled(NetDriver);
```

Comment:
Just realized that the previous setting defaulted to false, perhaps && would make more sense now?